### PR TITLE
June news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,670 @@
...
+## Package development corner
+
+Some useful tips for R package developers. :eyes:
+
+### Make your functions compa-tibble
+
+Do the functions of your package use `data.frame` as input?
+Do not miss Hugo Gruson's post [Make your functions compa-tibble](https://hugogruson.fr/posts/compa-tibble/) as users of your package might well try and pass a `tibble`, which you probably don't want to be a showstopper!
+
+### Use lintr to enforce your package's function preferences
```
Comment (steffilazerte) [inline content/blog/2024-06-21-news-june-2024/index.Rmd:632]: 🙀

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,670 @@
...
+Do you want to commit to using the [cli package](https://blog.r-hub.io/2023/11/30/cliff-notes-about-cli/) instead of base R messaging?
+You can configure the lintr settings for your codebase to pick up usage of certain functions, to inform you along with the preferred replacement.
+See, as an example, [pkgdown's lintr configuration file](https://github.com/r-lib/pkgdown/blob/819cede3fbbb2410912f4d49365177c9159b809f/.lintr.R#L1) and the corresponding [GitHub Actions workflow](https://github.com/r-lib/pkgdown/blob/main/.github/workflows/lint.yaml) (from [r-lib/actions](https://usethis.r-lib.org/reference/use_github_action.html)).
+This neat safeguard makes use of the [Undesirable function linter](https://lintr.r-lib.org/reference/undesirable_function_linter.html).
+
+### More metadata on CRAN
+
+CRAN pages of packages now show...
+
+- Issues needed fixing! [Shared by Henrik Bengtsson](https://mastodon.social/@henrikbengtsson/112591630316235677).
```
Comment (steffilazerte) [inline content/blog/2024-06-21-news-june-2024/index.Rmd:643]:
```suggestion
- Issues which need fixing! [Shared by Henrik Bengtsson](https://mastodon.social/@henrikbengtsson/112591630316235677).
```

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,670 @@
...
+
+- DOIs! Have you seen Achim Zeileis' announcement that [All R packages on CRAN will get an official DOI](https://fosstodon.org/@zeileis/112597049943483012)?
+Contrary to Zenodo DOIs, it's the same DOI across all versions.
```
Comment (steffilazerte) [inline content/blog/2024-06-21-news-june-2024/index.Rmd:646]: Softer language
```suggestion
In contrast to Zenodo DOIs, it's the same DOI across all versions.
```

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,670 @@
...
+
+
+### A pure GitHub preview workflow for pkgdown websites
```
Comment (steffilazerte) [inline content/blog/2024-06-21-news-june-2024/index.Rmd:649]: 🙀 🙀 🙀

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,670 @@
...
+
+### Tips for refactoring test files
+
+Do you put the object as close as possible to the related expectation(s)?
+Read about this, and other, [tips for refactoring test files](https://masalmon.eu/2024/05/23/refactoring-tests/).
+
+### One more tool for checking inputs of your R functions
+
+Do you check [inputs of your R functions](https://blog.r-hub.io/2022/03/10/input-checking/)?
+Beside the aforelinked R-hub blog post by Hugo Gruson, Sam Abbott, Carl Pearson, you might be interested in the experimental [stbl package](https://jonthegeek.github.io/stbl/) by Jon Harmon.
```
Comment (steffilazerte) [inline content/blog/2024-06-21-news-june-2024/index.Rmd:662]: Which blog post are you referring to?

**Review by steffilazerte — COMMENTED** [review]
Looks good @maelle!
I added the coworking for August and you blew my mind again with the development corner. 😁 

* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,678 @@
...
+
+- Two talks by rOpenSci team members, and more contributions by rOpenSci community members, to look forward to at useR! 2024 in Salzburg, Austria. 
+    - [Maëlle Salmon](/author/maëlle-salmon/)'s keynote talk ["How your code might rot"](https://sched.co/1c9Jy) on Wednesday, July 10 at 09:20 CEST;
+    - [Jeroen Ooms](/author/jeroen-ooms/)' talk  ["Navigating the R Ecosystem Using R-Universe"](https://sched.co/1c90h) on Thursday, July 11, 11:30 - 11:50 CEST.
+    - [Jon Harmon](/author/jon-harmon/)'s virtual talk "Learning Together at the Data Science Learning Community" will go live on the [useR!2024 YouTube playlist](https://youtube.com/playlist?list=PL77T87Q0eoJhsC203plZ1H4p21AGrM9rl&si=usuV_NlAJ_nR0QA8) at 10:30am CDT on July 2;
+    - [Paola Corrales](/author/pao-corrales/)' and [Elio Campitelli](/author/elio-campitelli/)'s tutorial ["Efficient Data Analysis with data.table"](https://sched.co/1c8yF) on Monday, July 3 (pre-registration required);
+    - [Elio Campitelli](/author/elio-campitelli)'s talk ["Building Bilingual Bridges with Multilingual Manuals"](https://sched.co/1c8yj) on Tuesday, July 4 at 11:40 CEST.
```
Comment (yabellini) [inline content/blog/2024-06-21-news-june-2024/index.Rmd:59]:
```suggestion
    - [Elio Campitelli](/author/elio-campitelli)'s talk ["Building Bilingual Bridges with Multilingual Manuals"](https://sched.co/1c8yj) on Tuesday, July 4 at 11:40 CEST.
    - [Lluís Revilla](/author/lluís-revilla-sancho/) and Henrik Bengtsson present a poster about CRAN packages archived and a the [cranhaven.org](http://cranhaven.org/) r-universe created to reduce the impact of that on users.
```

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,678 @@
...
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### PLACEHOLDER GOODPRACTICE MARK
+
+### PLACEHOLDER CZI MEETING YANI
```
Comment (yabellini) [inline content/blog/2024-06-21-news-june-2024/index.Rmd:48]:
```suggestion
### rOpenSci at CZI Open Science 2024

From June 10 to June 14 [Noam Ross](/author/noam-ross/), [Mauro Lepore](/author/mauro-lepore/) and [Yanina Bellini Saibene](/author/yanina-bellini-saibene/) participated on the [CZI Open Science 2024](https://chanzuckerberg.com/science/programs-resources/open-science/) event. 

On Wednesday, we showcase the [Champions Program](https://ropensci.org/champions/), sharing Champions' projects, training materials, and the results of the two-year pilot.  We have the chance to chat and learn about many other projects during these sessions. 

Yanina participated in the closing panel on _Case Study Session 3: Demonstrating Impact of Open Science_ to explore the challenges of using traditional academic metrics to measure project impact and emphasize alternative approaches. [In her talk](/events/czi-2024/), Yani introduced the work different rOpenSci members do, the tools and metrics we use to capture their stories, and the impact we achieve together.
```

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,670 @@
...
+
+### The rOpenSci community at events
+
+Meet rOpenSci team and community members at events in the near future!
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2024-06-21-news-june-2024/index.Rmd:58]: Added now!

File: content/blog/2024-06-21-news-june-2024/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,689 @@
...
+
+### PLACEHOLDER GOODPRACTICE MARK
+
+### rOpenSci at CZI Open Science 2024
+
+From June 10 to June 14 [Noam Ross](/author/noam-ross/), [Mauro Lepore](/author/mauro-lepore/) and [Yanina Bellini Saibene](/author/yanina-bellini-saibene/) participated on the [CZI Open Science 2024](https://chanzuckerberg.com/science/programs-resources/open-science/) event. 
+
+On Wednesday, we showcased the [Champions Program](https://ropensci.org/champions/), sharing Champions' projects, training materials, and the results of the two-year pilot.  We had the chance to chat and learn about many other projects during these sessions. 
+
+Yanina participated in the closing panel on _Case Study Session 3: Demonstrating Impact of Open Science_ to explore the challenges of using traditional academic metrics to measure project impact and emphasize alternative approaches. In [her talk](/events/czi-2024/), Yani introduced the work different rOpenSci members do, the tools and metrics we use to capture their stories, and the impact we achieve together.
```
Comment (mpadge) [inline content/blog/2024-06-21-news-june-2024/index.Rmd:54]:
```suggestion
Yanina participated in the closing panel on _Case Study Session 3: Demonstrating Impact of Open Science_ to explore the challenges of using traditional academic metrics to measure project impact and emphasize alternative approaches. In [her talk](/events/czi-2024/), Yani introduced the work done by different rOpenSci members, the tools and metrics we use to capture their stories, and the impact we achieve together.
```

**Pull request merged**